### ISSUE TITLE
Torrent does not start automatically with Tranmission 4.0.0

### ISSUE THREAD

**Issue by ZephireNZ** [Sonarr/Sonarr#5425]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Sonarr is configured with Tranmission and "Add Paused" is unchecked, however when torrents are added to Transmission they appear in the paused state.

### Expected Behavior

Torrents added to Transmission are automatically started if "Start Paused" is unchecked.

### Steps To Reproduce

1. Upgrade to transmission 4.0.0
2. Add torrent via Sonarr
3. Check Transmission UI to confirm torrent is in stopped state

### Environment

```markdown
- OS: ArchLinux 
- Sonarr: 3.0.9.1549 (linuxserver image)
- Transmission 4.0.0 (linuxserver image)
- Docker Install: Yes
- Using Reverse Proxy: Yes
- Browser:
```

### What branch are you running?

Main

### Trace Logs?

https://gist.github.com/ZephireNZ/fb4dc6d803b83a166dd341973f69e3de

**Comment by ckerr** [comment]
Yep. We're working on this upstream to fix for 4.0.1; don't blame Sonarr

**Comment by markus101** [comment]
Should be fixed in transmission 4.0.1.

**Comment by ckerr** [comment]
@markus101 a user is reporting this against 4.0.1 as well @ https://github.com/transmission/transmission/issues/5005. What is Sonarr's process for adding a torrent to Transmission?

**Comment by ckerr** [comment]
@bakerboy448 thanks for the fast response!

That code looks reasonable to me. I guess it's time for me to give Sonarr a testdrive to see if I can repro in 4.0.1

**Comment by ZephireNZ** [comment]
Can confirm this is working for me on 4.0.1 😊
